metaflow/plugins/cards/card_modules/basic.py

```diff
@@ -509,6 +510,7 @@ def render(self, task, stack_trace=None):
             javascript=JS_DATA,
             css=CSS_DATA,
             title=task.pathspec,
+            card_data_id=uuid.uuid4()
```

Comment:
in a previous PR we made ID optional, is it worth populating this for all cards so its not optional anymore?

Reply:
That would have to be a change that @valayDave makes. Default cards don't have ID's. Maybe they could?